src/js/main.js

```diff
@@ -142,8 +142,59 @@ $(document).ready(function() {
     });
   };
 
+  // 1. Create an array populated with people filtered first by category then by name
   if (bodyId === 'about-people') {
+    var people = [];
+    var categories = [
+      'founder',
+      'patron',
+      'board',
+      'ambassador',
+      'role-model',
+      'staff'
+    ];
+
+    // Do this for each category
+    for (var i = 0; i < categories.length; i++) {
+      var category = categories[i];
+      var peopleInCategory = [];
+      var $peopleInCategory = $('#people li.' + category);
+
+      // 1. Find all people of this category, then populate a temporary array
+      $peopleInCategory.each(function() {
+        peopleInCategory.push({
+          name: $(this).find('h2').text(),
+          role: $(this).find('h3').text(),
+          category: category
+        });
+      });
+
+      // 2. Sort each individual alphabetically by their name
+      var sortedPeopleInCategory = peopleInCategory.sort(function(a, b) {
+        if (a.name < b.name)
+          return -1;
+        if (a.name > b.name)
+          return 1;
+        return 0;
+      });
```

Comment:
you surprise me, I thought you might put this in one line ;)

Reply:
Hehe I admit, I copied this one from Stack Overflow.

Reply:
Here you go Sir `return (a > b) ? 1 : ((a < b) ? -1 : 0);` ;)